### PR TITLE
Adds canned iced coffee

### PR DESCRIPTION
**Pull request by Moltijoe**
# Why is this good for the game?
canned iced coffee

# Testing
image

# Spriting
image

:cl:
rscadd: Adds canned iced coffee
tweak: tweaks caffeine overdose to give warnings before starting to put you to sleep
imageadd: Adds canned iced coffee sprites
/:cl:

### CODE REVIEW

File: code/modules/food_and_drinks/drinks/drinks.dm [hunk 1]
```diff
@@ -522,10 +522,6 @@
 	list_reagents = list(/datum/reagent/consumable/lemon_lime = 30)
 	foodtype = FRUIT
 
-/obj/item/reagent_containers/food/drinks/soda_cans/lemon_lime/Initialize(mapload)
-	. = ..()
-	name = "lemon-lime soda"
```
Comment (ToasterBiome) [inline code/modules/food_and_drinks/drinks/drinks.dm]: move this up to the actual name if you're removing this part

**Review by ToasterBiome — CHANGES_REQUESTED** [review]
add COFFEE foodtype to other coffee related things

File: code/modules/food_and_drinks/drinks/drinks.dm (hunk 1 above)
Comment (Moltijoe) [reply code/modules/food_and_drinks/drinks/drinks.dm]: oh, this was a mistake

File: code/modules/food_and_drinks/drinks/drinks.dm [hunk 2]
```diff
@@ -525,7 +525,7 @@
 /obj/item/reagent_containers/food/drinks/soda_cans/lemon_lime/Initialize(mapload)
 	. = ..()
 	name = "lemon-lime soda"
-
+
```
Comment (ToasterBiome) [inline code/modules/food_and_drinks/drinks/drinks.dm]: !?!?!

**Review by ToasterBiome — CHANGES_REQUESTED** [review]
Still doesnt add COFFEE foodtype to other coffee drinks

File: code/modules/food_and_drinks/drinks/drinks.dm (hunk 2 above)
Comment (Moltijoe) [reply code/modules/food_and_drinks/drinks/drinks.dm]: die

**Review by ToasterBiome — CHANGES_REQUESTED** [review]
Readd COFFEE foodtype and add it to coffee like items

**Review by Mqiib — APPROVED** [review]
gud sprot

**Review by ToasterBiome — APPROVED** [review]
LGIM

**Pull request merged**